### PR TITLE
Test new model: Update withdraw liquidity test

### PR DESCRIPTION
**Pull request by amityadav0**

### CODE REVIEW

File: nearswap/src/lib.rs [hunk 1]
```diff
@@ -600,7 +600,7 @@ mod tests {
         _init(0)
     }
     fn init_with_storage_deposit() -> (Ctx, NearSwap) {
-        _init(NEP21_STORAGE_DEPOSIT * 120)
+        _init(NDENOM * 120)
```
Comment (robert-zaremba) [inline nearswap/src/lib.rs:603]: why we are charging 120 NAER here?

File: nearswap/src/lib.rs [hunk 2]
```diff
@@ -751,7 +751,7 @@ mod tests {
     fn add_liquidity2_happy_path() {
         let ynear_deposit = 30 * NDENOM;
         let token_deposit = 10 * NDENOM;
-        let ynear_deposit_with_storage = ynear_deposit + NEP21_STORAGE_DEPOSIT;
+        let ynear_deposit_with_storage = ynear_deposit + NDENOM;
```
Comment (robert-zaremba) [inline nearswap/src/lib.rs:754]: 1 NEAR is a lot here.

Comment (amityadav0) [reply nearswap/src/lib.rs:754]: yes decreasing

File: nearswap/src/lib.rs (hunk 1 above)
Comment (amityadav0) [reply nearswap/src/lib.rs:603]: decreasing

**Review by robert-zaremba — CHANGES_REQUESTED** [review]
It seams that we are missing tests for failures.

**Comment by amityadav0** [comment]
Added

**Review by robert-zaremba — APPROVED** [review]

**Pull request merged**